accounts/abi/bind/bind.go

```diff
@@ -594,6 +600,24 @@ func decapitalise(input string) string {
 	return strings.ToLower(goForm[:1]) + goForm[1:]
 }
 
+// convertToNil converts any type to its proper nil form.
+func convertToNil(input abi.Type) string {
+	switch input.T {
+	case abi.IntTy, abi.UintTy:
+		return "0"
+	case abi.StringTy:
+		return "\"\""
+	case abi.BoolTy:
+		return "false"
+	case abi.AddressTy:
+		return "0x0000000000000000000000000000000000000000"
+	case abi.HashTy:
+		return "0x0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Would it be cleaner to use `common.Address{}` and call the `String()` function so that it's more obviously correct?

Same comment for the hash type.